includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -382,4 +402,23 @@ private function getSitenameData() {
 			'value' => WikiFactory::getVarValueByName( 'wgSitename', $this->productInstanceId, false, $this->wg->Sitename ),
 		];
 	}
+
+	private function getRightSlot() {
```

Comment:
Is this "right" as in "left" vs. "right"? Are we going to regret calling it this when we want the same info but on the left side? Should this be called `extraGermanLogo`? J/K, but something to think about.